test/unittests/mongoBackend/mongoContextProvidersUpdate_test.cpp

```diff
@@ -1613,33 +1540,32 @@ TEST(mongoContextProvidersUpdateRequest, severalCprs)
   ASSERT_EQ(2, RES_CER(2).contextAttributeVector.size());
 
   EXPECT_EQ("A5", RES_CER_ATTR(2, 0)->name);
-  EXPECT_EQ("T", RES_CER_ATTR(2, 0)->type);
+  EXPECT_EQ("", RES_CER_ATTR(2, 0)->type);
   EXPECT_EQ(0, RES_CER_ATTR(2, 0)->value.size());
   EXPECT_EQ("http://cpr2.com", RES_CER_ATTR(2, 0)->providingApplication);
   EXPECT_TRUE(RES_CER_ATTR(2, 0)->found);
   EXPECT_EQ(0, RES_CER_ATTR(2, 0)->metadataVector.size());
 
   EXPECT_EQ("A6", RES_CER_ATTR(2, 1)->name);
-  EXPECT_EQ("T", RES_CER_ATTR(2, 1)->type);
+  EXPECT_EQ("", RES_CER_ATTR(2, 1)->type);
   EXPECT_EQ(0, RES_CER_ATTR(2, 1)->value.size());
   EXPECT_EQ("", RES_CER_ATTR(2, 1)->providingApplication);
   EXPECT_FALSE(RES_CER_ATTR(2, 1)->found);
   EXPECT_EQ(0, RES_CER_ATTR(2, 1)->metadataVector.size());
 
-  /* FIXME: how to mark that A6 is not found in the same Scc that is shared with A5? */
```

Comment:
At the end, I'm not marking the problem. I'm returning SccOk in the StatusCode for this element "passing the problem" to serviceRoutine. I think serviceRoutines is the best place to set the StatusCodes as it may depend (in some cases) of the result of the CPr forwarding, so StatusCode generated by mongoBackend are less imporant now. What do you think?